test/unit/lib/middleware/language.js

```diff
@@ -69,7 +69,7 @@ describe('language', function () {
 		});
 		var expected = 'zh-CN';
 		var req = mockRequest({
-			acceptLanguage: 'zh-CN;1,en-US;q=0.8'
+			acceptLanguage: 'zh-CN;q=1,en-US;q=0.8'
```

Comment:
Hey @frapan, sorry for the delay in getting back to you. The only thing I don't understand in your PR is the change to this file. Could you let me know what this is doing for us? I'd be happy to merge the other changes, just want to make sure I understand what's happening here.

Reply:
Thank you @blargity  for your reply and I'm sorry as well for being late.
I had an error during tests because of that line, and I couldn't go on with the tests.
In fact, the syntax of acceptLanguage expects the weights of the languages to be written with ";q=", not simply ";".
See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Language as a reference.